homeassistant/components/vesync/humidifier.py

```diff
@@ -107,15 +98,35 @@ class VeSyncHumidifierHA(VeSyncBaseEntity, HumidifierEntity):
     _attr_supported_features = HumidifierEntityFeature.MODES
 
     device: VeSyncHumidifierDevice
+    _ha_to_vs_mode_map: dict[str, str] = {}
+    _available_modes: list[str] = []
```

Comment:
Please don't create mutable objects on class level. These are shared between every instance of this object and can cause unwanted side effects